modules/openid_auth/app/controllers/openid_auth/v2/validation_controller.rb

```diff
@@ -11,7 +11,11 @@ class ValidationController < ApplicationController
       def index
         render json: validated_payload, serializer: OpenidAuth::ValidationSerializerV2
       rescue => e
-        raise Common::Exceptions::InternalServerError, e
+        if e.detail.eql?('Invalid request')
```

Comment:
Is this necessary? I don't recall seeing this pattern for other 4xx errors.

Reply:
The endpoint returned 500 if I didn't add the line.

Reply:
I took out the rescue altogether to address this.